src/overlays/actors/ovl_En_Ginko_Man/z_en_ginko_man.c

```diff
@@ -143,7 +143,8 @@ void EnGinkoMan_DepositDialogue(EnGinkoMan* this, GlobalContext* globalCtx) {
                 this->curTextId = 0x44E; //" ...So, what'll it be?  Deposit Rupees Don't deposit Rupees"
             } else {
                 func_801518B0(globalCtx, 0x44D, &this->actor);
-                this->curTextId = 0x44D; // "For example, if you deposit 200 Rupees, you'll get an item that holds a lot of Rupees."
+                this->curTextId =
+                    0x44D; // "For example, if you deposit 200 Rupees, you'll get an item that holds a lot of Rupees."
```

Comment:
Some of these long comments detailing exactly the text associated with the text id is causing some strange splits with clang. Perhaps this isn't the PR to address this, but perhaps we want to either shorten the comments or move them up or down a line to avoid these splits.

Reply:
I know in OoT they have a text table with all the text associated with their textIDs, so they've taken out all the comments of what the text is in overlay files and expect you to easily be able to look up what a textID is. I've been leaving in commented texts in MM since we don't have that textID table yet afaik (that could also be left out by design idk). But yeah it's worth bringing up, thanks

Reply:
OoT actually voted to delete the text table a few months ago, and have not replaced it by text extraction yet. I don't know how MM feels about this, but I'd expect erring on the side of not including.

Reply:
This is outside the scope of Angelo's PR, but yes. We will want to scrub the messages from merged code and extract the text table.